test/api/auth.js

```diff
@@ -70,21 +77,33 @@ test.serial('register: Email is taken', async (t) => {
   const res = await request(app)
     .post(REGISTER_ROUTE)
     .send({ email: 'testemail123456@mailinator.com', password: '123456789test1' });
+
   t.is(res.status, BAD_REQUEST);
 });
 
 test.serial('login: Success', async (t) => {
-  t.plan(4);
+  t.plan(8);
 
   const res = await request(app)
     .post(LOGIN_ROUTE)
     .send({ email: 'testemail123456@mailinator.com', password: '123456789test' });
 
-  refreshToken = res.body.data.refreshToken;
   t.is(res.status, SUCCESS);
   t.truthy(res.body.data);
   t.truthy(res.body.data.accessToken);
   t.truthy(res.body.data.refreshToken);
+
+  t.log('refresh:token: Success');
+
+  const res1 = await request(app)
+    .post(REFRESH_TOKEN_ROUTE)
```

Comment:
Wouldn't it be better if this was a separate test?

Reply:
I need logged in user in order to use his refresh token. Db is truncated after each test, that is why I merged them in 1 test.